### ISSUE TITLE
Weird signal processing output with no peaks and a double track pattern

### ISSUE THREAD

**Issue by brettviren** [WireCell/wire-cell-toolkit#133]
tl;dr: you need to flip the depo charge sign.

In particular, larsoft and WCT assume different electron sign conventions for "depos".  Check if the depo source component you use has a `scale` or similar configuration parameter and flip its sign.

Details:

You may see something like this simulation output of U plane ("orig" data tier). **Wrong:**

![okay](https://user-images.githubusercontent.com/209537/138110166-afc53812-e7a6-46b5-a8e7-24d0363284fd.png)

and it's corresponding SP output ("gauss" data tier).  **Wrong:**

![weird](https://user-images.githubusercontent.com/209537/138109725-3346eb0f-6f96-49ad-8dac-79b4efde51c0.png)

The SP output is obviously weird.  Less obvious is that the "orig" waveforms start out negative-going (time is from left to right).  Not shown here but the corresponding wrong sign "orig" for "W" is seen to be entirely negative.  

To be correct, bipolar "orig" should **start out positive** and unipolar "orig" should be **entirely positive**.

**Correct** U orig and gauss:

![u-orig](https://user-images.githubusercontent.com/209537/138110805-05dd642f-4982-4200-978c-42c5338bdbdd.png)
![u-gauss](https://user-images.githubusercontent.com/209537/138110834-2bb785c4-b3c8-4f23-a273-81d7e141d291.png)

**Correct** W orig and gauss:

![w-orig](https://user-images.githubusercontent.com/209537/138111132-68fdc8e5-53f8-420a-8d3b-a864bf12ed89.png)
![w-gauss](https://user-images.githubusercontent.com/209537/138111149-b2b24d2d-3f86-4a40-9b91-57aff8b0309b.png)
